extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderSpecialController.class.php

```diff
@@ -34,7 +34,11 @@ public function builder() {
 		$title = $this->getPar();
 		RenderContentOnlyHelper::setRenderContentVar( true );
 		RenderContentOnlyHelper::setRenderContentLevel( RenderContentOnlyHelper::LEAVE_GLOBAL_NAV_ONLY );
-		Wikia::addAssetsToOutput( 'portable_infobox_builder_scss' );
+
+		$this->response->addAsset( 'portable_infobox_builder_scss' );
+		$this->response->addAsset( 'portable_infobox_scss' );
+		$this->response->addAsset( 'portable_infobox_js' );
+
 		$url = implode( self::PATH_SEPARATOR, [ $this->wg->server, self::INFOBOX_BUILDER_MERCURY_ROUTE, $title ] );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)